functions/Get-DbaDbLogSpace.ps1

```diff
@@ -0,0 +1,139 @@
+
+    .EXAMPLE
+        PS C:\> Get-DbaDbLogSpace -SqlInstance Server1 -Database Database1, Database2
+
+        Returns the transaction log usage information for both Database1 and Database 2 on Server1
+
+    .EXAMPLE
+        PS C:\> Get-DbaDbLogSpace -SqlInstance Server1 -ExcludeDatabase Database3
+
+        Returns the transaction log usage information for all databases on Server1, except Database3
+
+    .EXAMPLE
+        PS C:\> Get-DbaDbLogSpace -SqlInstance Server1 -ExcludeSystemDatabase
+
+        Returns the transaction log usage information for all databases on Server1, except the system databases
+
+    .EXAMPLE
+        PS C:\> Get-DbaRegisteredServer -SqlInstance cmsServer | Get-DbaDbLogSpace -Database Database1
+
+        Returns the transaction log usage information for Database1 for a group of servers from SQL Server Central Management Server (CMS).
+
+    #>
+    [CmdletBinding()]
+    param (
+        [parameter(ValueFromPipeline, Mandatory)]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [string[]]$Database,
+        [string[]]$ExcludeDatabase,
+        [switch]$ExcludeSystemDatabase,
+        [switch]$EnableException
+    )
+
+    process {
+        foreach ($instance in $SqlInstance) {
+            try {
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential
+            } catch {
+                Stop-Function -Message "Failed to process Instance $Instance" -ErrorRecord $_ -Target $instance -Continue
+            }
+            $dbs = $server.Databases | Where-Object IsAccessible
+
+            if ($Database) {
+                $dbs = $dbs | Where-Object Name -in $Database
+            }
+            if ($ExcludeDatabase) {
+                $dbs = $dbs | Where-Object Name -NotIn $ExcludeDatabase
+            }
+
+            if ($ExcludeSystemDatabase) {
+                $dbs = $dbs | Where-Object IsSystemObject -eq $false
+            }
+
+            # 2012+ use new DMV
+            if ($server.versionMajor -ge 11) {
+                foreach ($db in $dbs) {
+                    try {
+                        $logspace = $server.query('select * from sys.dm_db_log_space_usage', $db.name)
+                    } catch {
+                        Stop-Function -Message "Unable to select from sys.dm_db_log_space_usage on $instance." -ErrorRecord $_ -Target $db -Continue
+                    }
+                    [pscustomobject]@{
+                        ComputerName        = $server.ComputerName
+                        InstanceName        = $server.ServiceName
+                        SqlInstance         = $server.DomainInstanceName
+                        Database            = $db.name
+                        LogSize             = [dbasize]($logspace.total_log_size_in_bytes)
+                        LogSpaceUsedPercent = $logspace.used_log_space_in_percent
```

Comment:
Random tidbit, that we don't have to put here...but did y'all know there is an overload for `.ToString()` where you can actually make it include the `%` sign on the output?

```
[PS 6.2.2] [08:53:00] > C:\Users\Shawn Melton
[1] > (45/254).ToString("P")
17.72%
```

Reply:
I did not know that, should I add it?

Reply:
i dont think so, unless % is ignored in calculations

Reply:
No, not to be added to our code...this is used for when you are ready to just display the data.

Reply:
I got excited.... removed it :)